src/docfx.website.themes/default/partials/class.header.tmpl.partial

```diff
@@ -50,7 +50,7 @@
 {{#syntax.parameters}}
     <tr>
       <td>{{{type.specName.0.value}}}</td>
-      <td><em>{{{id}}}</em></td>
+      <td><span class="parametername">{{{id}}}</span></td>
```

Comment:
><span class="parametername"> [](start = 10, length = 28)

this is breaking change for default template, make sure to mention this change in our release note